BrainCloudClient/src/BrainCloud/Client/BrainCloud/Internal/FileUploader.cs

```diff
@@ -92,14 +92,19 @@ public enum FileUploaderStatus
         private CancellationTokenSource _cancelToken;
 #endif
 
+        private BrainCloudClient client;
```

Comment:
i see all private variables are being prefixed with _ instead of m_ ... but lets stick to ONE format

_

ie, in this case client would be _client since its private